upi/vsphere/README.md

```diff
@@ -1,59 +1,47 @@
-# Pre-Requisites
+# Installing OpenShift on VMware
 
-* terraform
-* jq
+The terraform examples provided require a AWS account.  AWS is used
+for its API interface to network load balancing and DNS.
+The `./modules/rhcos_virtual_machines` can be used independently if so desired.
 
-# Build a Cluster
+We will begin with using `openshift-installer` as each terraform example 
+starts with creating the ignition files.
 
-1. Create an install-config.yaml.
-The machine CIDR for the dev cluster is 139.178.89.192/26.
+## Create ignition files for OpenShift RHCOS virtual machines
+
+You can create a `install-config.yaml` file by copying the example provided:
 
 ```
-apiVersion: v1
-baseDomain: devcluster.openshift.com
-metadata:
-  name: mstaeble
-networking:
-  machineCIDR: "139.178.89.192/26"
-platform:
-  vsphere:
-    vCenter: vcsa.vmware.devcluster.openshift.com
-    username: YOUR_VSPHERE_USER
-    password: YOUR_VSPHERE_PASSWORD
-    datacenter: dc1
-    defaultDatastore: nvme-ds1
-pullSecret: YOUR_PULL_SECRET
-sshKey: YOUR_SSH_KEY
+cd ./upi/vsphere/
+cp install-config.yaml{.example,}
```

Comment:
nit: brace expansion is [a Bashism][1].  We'd be portable across all POSIX shells with:

```sh
cp install-config.yaml.example install-config.yaml
```

[1]: https://mywiki.wooledge.org/Bashism